linux_docker_resources/Dockerfile

```diff
@@ -8,17 +8,18 @@ ARG PLATFORM=x86
 # See: http://askubuntu.com/questions/506158/unable-to-initialize-frontend-dialog-when-using-ssh
 ENV DEBIAN_FRONTEND noninteractive
 
+RUN (apt-get update || true) && apt-get install -y locales
```

Comment:
Yeah, that `apt-get update || true` is not ideal since it also hides other kinds of update failures too. I have the suspicion that this will come back to bite us, but if you don't see a straightforward way to avoid the docker hub issue, which you described in a comment, then this the best solution.